ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -122,6 +123,13 @@ public async Task<Notification> GetNotificationWithImmunosuppresionDetailsAsync(
                 .FirstOrDefaultAsync(n => n.NotificationId == NotificationId);
         }
 
+        public async Task<Notification> GetNotificationWithDenotificationDetailsAsync(int notificationId)
+        {
+            return await GetBaseNotificationIQueryable()
+                .Include(n => n.DenotificationDetails)
```

Comment:
We shouldn't need this class, owned models are automatically included by EF! There are a few other redundant ones (Immuno, SocialRisk) but I've already just removed them in my latest commit so no need to worry about those :)